js/app/packages/app/index.css

```
@@ -125,16 +120,32 @@
   --color-ink-placeholder: var(--c4);
 
+  /* Below are our semantic surface tokens, which either pick up color overrides from the theme, or are set to --b0, which allows them to be controlled by our Layer system */
+  /* NOTE: these are NOT all wired up. We erased a lot of our semantic design tokens when moving to Layer system, these are part of a migration back towards semantic tokens. */
+  /* To wire them up you should change the generic '*-surface' class on a component to instead use one of these tokens. */
+  --color-page:            var(--theme-page, var(--b0));
+  --color-panel:           var(--theme-panel, var(--b0));
+  --color-inset:           var(--theme-inset, var(--b0));
+  --color-dialog:          var(--theme-dialog, var(--b0));
+  --color-menu:            var(--theme-menu, var(--b0));
+  --color-input:           var(--theme-input, var(--b0));
+  --color-message:         var(--theme-message, color-mix(in oklch, var(--b1) 50%, var(--b2)));
```

Stylelint (17.14.0)

[error] Duplicate custom property "--color-message" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)